cmd/harmony/config_migrations.go

```diff
@@ -156,4 +157,14 @@ func init() {
 		confTree.Set("Version", "2.0.0")
 		return confTree
 	}
+
+	migrations["2.0.0"] = func(confTree *toml.Tree) *toml.Tree {
```

Comment:
Have you tested running config of version v1.0.4 or earlier? Is there a upgrade logic from v1.0.4 -> v2.0.0 -> v2.1.0?

Reply:
Hi @JackyWYX  
I believe that's covered by `cmd/harmony/config_migrations_test.go`